source/class/qx/ui/toolbar/Button.js

```diff
@@ -19,32 +19,12 @@
 
 /**
  * The normal toolbar button. Like a normal {@link qx.ui.form.Button}
- * but with a style matching the toolbar and without keyboard support.
+ * but with a style matching the toolbar.
```

Comment:
Behavior change

Reply:
Needed. The button is not reachable for keyboard users otherwise